<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>categoryAdd</class>
 <widget class="QWidget" name="categoryAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="title">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Wheel of Jeopardy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="category">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="category_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Category Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="question1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="q1_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Question $200:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="a1_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer $200:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="question2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="a2_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer $400:</string>
   </property>
  </widget>
  <widget class="QLabel" name="q2_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Question $400:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="question3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="a3_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer $600:</string>
   </property>
  </widget>
  <widget class="QLabel" name="q3_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Question $600:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="question4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="a4_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer $800:</string>
   </property>
  </widget>
  <widget class="QLabel" name="q4_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Question $800</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="question5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>580</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="a5_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer $1,000:</string>
   </property>
  </widget>
  <widget class="QLabel" name="q5_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Question $1,000:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>633</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
